### ISSUE TITLE
MinMax Scaler Bug with Predict Method

### ISSUE THREAD

**Issue by PotosnakW** [Nixtla/neuralforecast#671]
### What happened + What you expected to happen

There may be a bug with the 'minmax' scaler option. When I train Auto models with this scaler and use the saved models to manually generate predictions using the 'predict' method, I get 'NaN' value predictions. However, the csv file prediction output of the Auto model cross validation method generates a dataframe with actual predictions (non-Nan values). 

### Versions / Dependencies

- OS: Springdale Open Enterprise Linux (--> based on Red Hat Enterprise Linux)
- Browser: N/A
- Version: 8.7 Moderna

### Reproduction script

import torch
import pandas as pd
from datasetsforecast.long_horizon import LongHorizon
from ray import tune
from neuralforecast.auto import AutoNHITS
from neuralforecast.core import NeuralForecast
from neuralforecast.losses.pytorch import MSE
import os
import logging
logging.getLogger("pytorch_lightning").setLevel(logging.WARNING)

Y_df, _, _ = LongHorizon.load(directory='./', group='ETTm2')
Y_df['ds'] = pd.to_datetime(Y_df['ds'])

n_time = len(Y_df.ds.unique())
val_size = int(.2 * n_time)
test_size = int(.2 * n_time)
horizon = 96
input_size = 192

nhits_config0 = {
       "learning_rate": tune.choice([1e-3]),                                     
       "max_steps": tune.choice([10]),                                           
       "val_check_steps": tune.choice([5]),
       "input_size": tune.choice([2 * horizon]),                                
       "batch_size": tune.choice([7]),                                      
       "windows_batch_size": tune.choice([32]),  
       "scaler_type": tune.choice([None]),      
       "random_seed": tune.randint(1, 10),
    }
nhits_config1 = {
       "learning_rate": tune.choice([1e-3]),                                     
       "max_steps": tune.choice([10]),                                           
       "val_check_steps": tune.choice([5]),
       "input_size": tune.choice([2 * horizon]),                                
       "batch_size": tune.choice([7]),                                      
       "windows_batch_size": tune.choice([32]),  
       "scaler_type": tune.choice(['robust']),      
       "random_seed": tune.randint(1, 10),
    }
nhits_config2 = {
       "learning_rate": tune.choice([1e-3]),                                    
       "max_steps": tune.choice([10]),                                       
       "val_check_steps": tune.choice([5]),
       "input_size": tune.choice([2 * horizon]),                                 
       "batch_size": tune.choice([7]),                                           
       "windows_batch_size": tune.choice([32]),  
       "scaler_type": tune.choice(['minmax']),      
       "random_seed": tune.randint(1, 10),
    }
nhits_config3 = {
       "learning_rate": tune.choice([1e-3]),                                    
       "max_steps": tune.choice([10]),                                       
       "val_check_steps": tune.choice([5]),
       "input_size": tune.choice([2 * horizon]),                                 
       "batch_size": tune.choice([7]),                                           
       "windows_batch_size": tune.choice([32]),  
       "scaler_type": tune.choice(['minmax1']),      
       "random_seed": tune.randint(1, 10),
    }
    
models = [AutoNHITS(h=horizon,
                    loss=MSE(),
                    config=nhits_config0,
                    num_samples=5),
          AutoNHITS(h=horizon,
                    loss=MSE(),
                    config=nhits_config1,
                    num_samples=5),
          AutoNHITS(h=horizon,
                    loss=MSE(),
                    config=nhits_config2,
                    num_samples=5),
          AutoNHITS(h=horizon,
                    loss=MSE(),
                    config=nhits_config3,
                    num_samples=5),
         ]

os.makedirs('./test_results', exist_ok=True)

fcst = NeuralForecast(models=models, freq='15min')
fcst_df = fcst.cross_validation(df=Y_df, val_size=val_size,test_size=test_size, n_windows=None)
fcst.save(path='./test_results/',
          model_index=None,
          overwrite=True,
          save_dataset=False)
fcst_df.to_csv('./test_results/forecasts.csv', index=False)

model = NeuralForecast.load(path= './test_results/')
df_test = Y_df[Y_df.unique_id == 'OT']
df_test = df_test.iloc[-(test_size+input_size):]
y_pred_df = df_test.iloc[0:input_size]
y_hat_df = model.predict(df=y_pred_df)
y_hat_df.to_csv('./test_results/model_forecasts.csv')

### Issue Severity

High: It blocks me from completing my task.
